alignCom.html

```diff
@@ -0,0 +1,67 @@
+//This<!DOCTYPE html>
+<html>
+<head>
+  <title>Align Commerce JS Test</title>
+
+  <style>
+  div{
+    margin-top: 30%;
+    position: relative;
+    bottom: 45%;
+    left: 35%; 
+    font-size: 3rem;
+  }
+
+  </style>
+</head>
+	<body>
+    <div>
+      Backgrond color changed for: <span id="text"></span>
+    </div>
+  </body>
+    
+</html>
+
+<script type="text/javascript">
+	(function(){
+		'use strict'
+		var body = document.body;
+    var text = document.getElementById('text');
+		var usedColor = {};
+		
+		var timer = 1;
+
+    var getRandomNum = function(){
+      return Math.round(Math.random() * 255);
+    }
+
+    var formNumArr = function(){
```

Comment:
can we rename this function since I dont really know what it means unless i read the code inside.  Maybe something like `generateRandomNumbers`?